exercises/06-deploy-to-cf.md

```diff
@@ -32,56 +44,65 @@ This is what the service instances should look like in the SAP BTP cockpit:
 ![SCP Services](images/scp-services.png)
 
 ## Adjust the manifest.yml file
+
 Let's adjust the [manifest.yml](../manifest.yml), the configuration file for your Cloud Foundry application.
-Take a look at the services section, where we reference the previously created services instances.
-Replace all occurrences of `<participantId>` in the manifest.yml file.
+Take a look at the services section, where we reference the previously created service instances.
```

Comment:
[pp] I think there is no adjustment needed? The names are already there

Reply:
yes. I removed this.